heartbeat/docs/fields.asciidoc

```diff
@@ -128,20 +128,6 @@ None
 
 
 [float]
-=== @timestamp
-
-type: date
-
-example: 2015-01-24 14:06:05.071000
-
-format: YYYY-MM-DDTHH:MM:SS.milliZ
```

Comment:
@urso Was there a reason you specified the format here instead of just using date?